Sources/xcproj/XCWorkspace.swift

```diff
@@ -25,6 +25,14 @@ public class XCWorkspace {
         }
         data = try XCWorkspace.Data(path: xcworkspaceDataPaths.first!)
     }
+    
+    /// Initializes the workspace with the path string.
+    ///
+    /// - Parameter pathString: path string.
+    /// - Throws: throws an error if the initialization fails.
+    public convenience init(pathString: String) throws {
```

Comment:
Looks good; 2 things though.

1. The fact that `init(path: Path)` is still part of the `API` and public wouldn't still require users to import `Path` in order to properly compile/use the tool?
1. I'm assuming that `Path` checks if a `String` is a valid path or not but perhaps we could check in the `convenience init`s if the passed `String` is a valid path and if not `throw` with a proper/more meaningful error?

Reply:
1. A user wouldn't need to import `Path` if they are not making reference to the symbol, so this is fine.

Reply:
Thanks @yonaskolb wasn't 💯  sure it would work or not.